mobile-app/app/screens/AppNavigator/screens/Loans/components/LoanCardsV2.tsx

```diff
@@ -231,3 +288,39 @@ function LoanCard({
     </ThemedViewV2>
   );
 }
+
+/* 
+  Custom comparison is added because the debounced search is sluggish due to Loan Icon rendering
+*/
+const MemoizedLoanIcon = memo(
+  ({ testID, displaySymbol }: { testID: string; displaySymbol: string }) => {
+    const LoanIcon = getNativeIcon(displaySymbol);
+    return (
+      <View
+        style={tailwind("flex flex-row justify-between items-center w-full")}
+      >
+        <ThemedTextV2
+          light={tailwind("text-mono-light-v2-700")}
+          dark={tailwind("text-mono-dark-v2-700")}
+          style={tailwind("font-medium font-normal-v2")}
```

Comment:
wont the font-normal-v2 override the font-medium?

```suggestion
          style={tailwind("font-normal-v2")}
```